docs/src/main/sphinx/release/release-449.md

```diff
@@ -35,9 +35,10 @@
 ## Hive connector
 
 * Add support for specifying a catalog name in the Thrift metastore with the
-  `hive.metastore.thrift.catalog-name` configuration property. (`10287`)
-* Add support for configuring the query retry policy on the S3 filesystem with
-  the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+  `hive.metastore.thrift.catalog-name` configuration property. ({issue}`10287`)
+* Add support for configuring the query retry policy on the native S3 filesystem
+  with the `s3.retry-mode` and `s3.max-error-retries` configuration properties.
+  ({issue}`21900`)
 * Improve performance of writing to Parquet files. ({issue}`22089`)
 * Fix failure when filesystem caching is enabled on Trino clusters with a single
```

Comment:
Please see https://github.com/trinodb/trino/pull/22195#discussion_r1623137488 
We didn't fix any explicit failure here

Reply:
Updated